index.js

```diff
@@ -20,28 +31,50 @@ function renderWords() {
     for (let i = 0; i < game.words.length; i++) {
         const spanElement = document.createElement("span");
         spanElement.setAttribute("id", i);
+        spanElement.setAttribute("class", "fs-4");
         spanElement.innerHTML = game.words[i];
         document.getElementById("random-words").appendChild(spanElement);
     }
 }
 
+function derenderWords() {
+    for (let i = 0; i < game.words.length; i++) {
+        const item = document.getElementById(i);
```

Comment:
Missing comments. What is the purpose of this function?

Reply:
Updated.